File: app/src/main/java/com/nollpointer/dates/SortCardsControl.java
package com.nollpointer.dates;

import java.util.List;

public interface SortCardsControl{
    void refresh();
    void setQuestions(List<String> list);
    void setAnswerSequence(int[] sequence);
    boolean check();
    //void setOnCardClickListener();
    void setResultScreen();
    int getAnswerSequence();

    void setCheckMode(boolean state);
}

File: app/src/main/java/com/nollpointer/dates/ui/sort/SortCardsControl.kt
package com.nollpointer.dates.ui.sort

/**
 * @author Onanov Aleksey (@onanov)
 */
interface SortCardsControl {
    fun setQuestions(list: List<String?>?)
    fun setAnswerSequence(sequence: IntArray?)
    fun check(): Boolean
    fun setCheckMode(state: Boolean)
    fun setColors(g: Int, r: Int)
}